<documentation title="Disallow Short Array Syntax">
    <standard>
    <![CDATA[
    The array keyword must be used to define arrays.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Using long form array syntax.">
        <![CDATA[
$arr = <em>array(</em>
    'foo' => 'bar',
<em>)</em>;
        ]]>
        </code>
        <code title="Invalid: Using short array syntax.">
        <![CDATA[
$arr = <em>[</em>
    'foo' => 'bar',
<em>]</em>;
        ]]>
        </code>
    </code_comparison>
</documentation>
