<documentation title="Mixed Keyed Unkeyed Array">
    <standard>
    <![CDATA[
        All items in an array should either have an explicit key assigned or none. A mix of keyed and unkeyed items is not allowed.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Arrays with all items either keyed or unkeyed.">
        <![CDATA[
$args = array(
    <em>'foo'</em> => 22,
    <em>'bar'</em> => 25,
);

$args = array(22, 25);
        ]]>
        </code>
        <code title="Invalid: Array with a mix of keyed and unkeyed items.">
        <![CDATA[
$args = array(
    'foo' => 22,
    25,
);
        ]]>
        </code>
    </code_comparison>
</documentation>
