<?xml version="1.0"?>
<documentation title="Lowercase Class Resolution Keyword">
    <standard>
    <![CDATA[
    The "class" keyword when used for class name resolution, i.e. `::class`, must be in lowercase.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: using lowercase.">
        <![CDATA[
echo MyClass::class;
        ]]>
        </code>
        <code title="Invalid: using uppercase or mixed case.">
        <![CDATA[
echo <em>MyClass::CLASS</em>;
        ]]>
        </code>
    </code_comparison>
</documentation>
